snapcraft.yaml

```diff
@@ -31,6 +31,7 @@ parts:
       - gdisk
       - mtools
       - python3-docutils
+      - patchelf
```

Comment:
Doesn't snapcraft bundle its own copy of patchelf? Or did they stop that?

Reply:
Right, this is not needed. I was invoking manually patchelf until I noticed that it is still supported by snapcraft, but now it is opt-in instead of opt-out.